src/scripts/main.js

```
@@ -1,3 +1,18 @@
 'use strict';
 
 // write your code here
+const populationElements = document.querySelectorAll('.population');
+
+const populations = Array.from(populationElements)
+  .map((span) => span.textContent.replace(/,/g, ''))
```

This line `.map((span) => span.textContent.replace(/,/g, ''))` only removes commas. The task requires handling existing thousands separators and ensuring strings are convertible to numbers. Consider removing any grouping characters (commas, non-breaking spaces, thin spaces, etc.) or using a regex to keep only digits, optional decimal point and sign (for example: `.replace(/[^\d.-]/g, '')`). That will avoid silently failing for other separator styles.